independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateGlobal.java

```diff
@@ -7,6 +7,10 @@
 import java.lang.annotation.Target;
 
 /**
+ * <p>
+ * <strong>IMPORTANT: This annotation only works in a fully integrated environment; such as a Quarkus application.</strong>
```

Comment:
I can see this _explained_ (noted) in https://quarkus.io/guides/qute-reference#standalone about `@TemplateData`, `@TemplateExtension` and `@TemplateEnum`. Can you add there also `@TemplateGlobal`, please? If I am using Qute as standalone library, I should be instructed how to workaround this or understand limitations. Thank you

Reply:
Done.